specification.md

```diff
@@ -406,6 +416,33 @@ Error message, typically human readable.
 
 Additional [RetryInfo](#throttling) message about retrying if type==UNAVAILABLE.
 
+<h2 id="servertoagentcommand-message">ServerToAgentCommand Message</h2>
+ 
+
+The message has the following structure:
+
+```protobuf
+// ServerToAgentCommand is sent from the server to the agent to request that the agent
+// perform a command.
+message ServerToAgentCommand {
+    enum CommandType {
+        // The agent should restart. This request will be ignored if the agent does not
+        // support restart.
+        Restart = 0;
+
+        // The agent should shutdown. This request will be ignored if the agent does not
+        // support shutdown. Shutdown is permanent and the agent will no longer be running
+        // or connected to the management server.
+        Shutdown = 1;
```

Comment:
I thought a bit more about this and it is not quite clear how this can be implemented by an Agent. Typically the Agent is installed in a way that autostarts it on system start, right? So do we expect the shutdown to disable the autostart? Is this even doable in typical use-cases when the user that the Agent runs with is not the root?